subsys/bluetooth/services/hrs.c

```diff
@@ -91,6 +101,34 @@ static int hrs_init(void)
 	return 0;
 }
 
+int bt_hrs_cb_register(struct bt_hrs_cb *cb)
+{
+	if (cb == NULL) {
+		return -EINVAL;
+	}
+
+	sys_slist_append(&hrs_cbs, &cb->node);
+
+	return 0;
+}
+
+int bt_hrs_cb_unregister(struct bt_hrs_cb *cb)
+{
+	if (cb == NULL) {
+		return -EINVAL;
+	}
+
+	if (sys_slist_is_empty(&hrs_cbs)) {
+		return -ENOENT;
+	}
```

Comment:
This is superfluous, right?

Reply:
I'm not sure what you mean. How will the code look like after the change you have in mind?

Reply:
Never mind ;-)I'll remove the `if` statement.